pkg/pillar/cmd/nim/nim.go

```diff
@@ -107,8 +107,12 @@ func Run() {
 	}
 	defer logf.Close()
 	if nimCtx.useStdout {
-		multi := io.MultiWriter(logf, os.Stdout)
-		log.SetOutput(multi)
+		if logf == nil {
```

Comment:
This part I don't get -- why are we giving nim the same treatment we're giving to diag/client (which are not long running processes)?

Reply:
I don't recall the history for that. I just fixed the code which did a multiwriter where one of the writers is nil, which is a bit odd.